var/spack/repos/builtin/packages/py-mizani/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.directives import depends_on, version
+
+
+class PyMizani(PythonPackage):
+    """Mizani is a scales package for graphics. It is based on Hadley Wickham's
+    Scales package."""
+
+    pypi = "mizani/mizani-0.7.3.tar.gz"
+
+    version(
+        "0.7.3",
+        sha256="f521300bd29ca918fcd629bc8ab50fa04e41bdbe00f6bcf74055d3c6273770a4",
+    )
+
```

Comment:
```python
depends_on('python@3.6:', type=('build', 'run'))
```

Reply:
Done in 41dd0358424c3cd75acba4ee5226381a22fb14d3.